### DISCUSSION TITLE
Tenerife map

### DISCUSSION THREAD

**Discussion by zwannie** [mapsforge/mapsforge#1566] [Q&A] [answered]
Where can i find the Tenerife map? It is not at the Spain map as i expected.

**Comment by devemux86** [comment, marked as answer]
Mapsforge maps use the regions as defined in the [Geofabrik OpenStreetMap Data Extracts](https://download.geofabrik.de/africa/canary-islands.html).

So Canary Islands map with Tenerife is located in the [Africa folder](https://ftp-stud.hs-esslingen.de/pub/Mirrors/download.mapsforge.org/maps/v5/africa/).
